tests/helpers/acceptance.js

```diff
@@ -55,6 +58,18 @@ function createTestTargets(projectName, options) {
   options.command = options.command || 'new';
 
   return applyCommand(options.command, projectName, '--skip-npm', '--skip-bower', `--directory=${outputDir}`)
+    .then(() => {
+      if (options.createESLintConfig) {
```

Comment:
I'd prefer that too, but I'm not sure how to do it as long as we use `--skip-npm` here :-/